app/controllers/ProductController.scala

```diff
@@ -15,20 +16,26 @@ import javax.inject.Inject
 class ProductController @Inject() (
                                     cc: ControllerComponents,
                                     environment: Environment
-                                  ) extends AbstractController(cc) {
-  // Create the format for the product in JSON form.
-  implicit val productFormat = Json.format[Product]
+                                  ) extends AbstractController(cc) with ProductDAO {
 
   // When the user route to <root>/pass-bakery/status then this will return a status JSON
-  def getStatus = Action {
+  def getStatus: Action[AnyContent] = Action {
 
     // Used LocalDateTime since it time and date is formatted in ISO-8601 as it is written in its doc.
     val curDateTime = LocalDateTime.now()
 
     val curEnv = environment.mode
-    val product = Product("pass-bakery", curEnv.toString, curDateTime.toString)
+    val status = EndPointStatus("pass-bakery", curEnv.toString, curDateTime.toString)
 
     // Send OK signal with JSON status
-    Ok(Json.toJson(product))
+    Ok(Json.toJson(status))
+  }
+
+  // When the user route to <root>/rest/bakery/products then this will return all the elements of the bakery
+  // database.
+  def getInventory: Action[AnyContent] = Action {
```

Comment:
```suggestion
  /** 
   * When the user route to <root>/rest/bakery/products then this will return all the elements of the bakery
   * database.
  */
  def getInventory: Action[AnyContent] = Action {
```

https://docs.scala-lang.org/style/scaladoc.html

Reply:
I did change it to that, thanks!